CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+Contributions of any kind are most welcome! Just open an [issue](https://github.com/PROGRADE-Tech/Tarchivebot/issues/new) or send a [pull request](https://github.com/PROGRADE-Tech/Tarchivebot/compare). All code contributions will be licensed under the [GNU AGPL-3.0](https://github.com/PROGRADE-Tech/Tarchivebot/blob/master/LICENSE).
+
+## Pull requests
+
+You can use whatever tool you feel most comfortable with to develop your feature. However we do encourage you to use the Git command line interface. Here are the steps to submitting a PR using the CLI:
+
+1. Click the fork button on the [Tarchivebot repository](https://github.com/PROGRADE-Tech/Tarchivebot).
+2. Clone your fork to your local machine: `git clone https://github.com/{{your_username}}/Tarchivebot.git && cd Tarchivebot`
+3. Create your branch and name it after the related issue or feature `git checkout -b {{feature_name}}`
+4. Now you can start coding your feature and changes.
+5. After you're done commit your changes. `git commit -am "{{commit_message}}"`. Signed commits are also very much appreciated: `git commit -S -m "{{commit_message}}"`.
+6. Push your changes to your fork: `git push origin -b {{feature_name}}`
+7. Go to your fork on GitHub and create the pull request!
+
+## Style guidelines
+
+We're not at all strict about any particular coding standard. However you can take a look at [Javascript Standard Style](https://github.com/feross/standard/blob/master/RULES.md) if you need a general styling or formatting guideline to adhere to.
+
```

Comment:
Maybe add a notice that in future StandardJS will be enforced and that especially new files should adhere to that